app/index.js

```diff
@@ -27,7 +27,7 @@ export default class InfiniteScroll extends Component {
   }
 
   componentDidMount () {
-    this.el = this.props.height ? this._infScroll : window;
+    this.el = this.props.height ? this._infScroll : this.props.ancestorScroll ? this.props.ancestorScroll : window;
```

Comment:
can you please rename this to `scrollableTarget` (I feel this to be more generic, but I've no problems with ancestorScroll also). Also we can remove the ternary using
```js
this.el = this.props.height ? this._infScroll : this.props.scrollableTarget || window;
```